grpcgcp/grpc_gcp/grpc_gcp.proto

```diff
@@ -17,21 +17,26 @@ syntax = "proto3";
 package grpc.gcp;
 
 message ApiConfig {
-  // The channel pool configurations.
+  // The channel/connection pool configurations.
   ChannelPoolConfig channel_pool = 2;
 
   // The method configurations.
   repeated MethodConfig method = 1001;
 }
 
 message ChannelPoolConfig {
-  // The max number of channels in the pool.
+  // The max number of channels/connections in the pool.
+  // Default value is 0, meaning 'unlimited' size.
   uint32 max_size = 1;
-  // The idle timeout (seconds) of channels without bound affinity sessions.
+
+  // The idle timeout (seconds) of channels/connections without bound affinity sessions.
   uint64 idle_timeout = 2;
+
   // The low watermark of max number of concurrent streams in a channel.
-  // New channel will be created once it get hit, until we reach the max size
-  // of the channel pool.
+  // New channel/connection will be created once it get hit, until we reach
+  // the max size of the channel/connection pool.
+  // Default value is 100, and the valid range is [1, 100], any value greater
+  // than 100 will be rounded down to 100.
```

Comment:
I would consider adding a newline that says:

Note: It is not recommended that users adjust this value, since a single channel should generally have no trouble managing the default (maximum) number of streams.

Reply:
Done.